packages/design-system/src/components/text-field.tsx

```diff
@@ -129,19 +141,72 @@ export const TextField = styled("input", {
         },
       },
     },
-    cursor: {
-      default: {
-        cursor: "default",
-        "&:focus": {
-          cursor: "text",
-        },
-      },
-      text: {
-        cursor: "text",
-      },
-    },
   },
   defaultVariants: {
     size: "1",
   },
 });
+
+type TextFieldProps = Omit<
+  React.ComponentProps<typeof TextFieldBase> & React.ComponentProps<"input">,
+  "prefix"
```

Comment:
An alternative fix for [the one posted below](https://github.com/webstudio-is/webstudio-designer/pull/292/files#r975490001). Perhaps this one is even better

```suggestion
  "prefix" | "ref"
```